apps/docs/src/app/core/component-docs/moment-datetime-adapter/examples/moment-datetime-formats-example.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
```

Comment:
In the docs for some reason instead of providing that code as example, `-options-` example is provided.

![image](https://user-images.githubusercontent.com/20265336/114711946-381baf80-9d38-11eb-9852-837d45c30e03.png)